src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -0,0 +1,450 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import analysis.normed_space.basic
+import topology.sequences
+
+/-!
+# Normed groups homomorphisms
+
+This file gathers definitions and elementary constructions about bounded group homomorphisms
+between normed (abelian) groups (abbreviated to "normed group homs").
+
+The main lemmas relate the boundedness condition to continuity and Lipschitzness.
+
+The main construction is to endow the type of normed group homs between two given normed groups
+with a group structure and a norm, giving rise to a normed group structure.
+
+Some easy other constructions are related to subgroups of normed groups.
+-/
+noncomputable theory
+open_locale nnreal big_operators
+
+set_option old_structure_cmd true
+
+/-- A morphism of normed abelian groups is a bounded group homomorphism. -/
+structure normed_group_hom (V W : Type*) [normed_group V] [normed_group W] extends V →+ W :=
+(bound' : ∃ C, ∀ v, ∥to_fun v∥ ≤ C * ∥v∥)
```

Comment:
If you use `old_structure_cmd` anyway (BTW, is there any reason?), then you can safely omit `map_zero'` from axioms.

Reply:
You could also add:
```lean
(map_zero := sorry)
```
to pre-fill the axiom

Reply:
Thanks, fixed.